### DISCUSSION TITLE
Inactive in less than five minutes

### DISCUSSION THREAD

**Discussion by Storinob** [RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK#55] [Ideas]
Can you add the ability to specify the time before switching to the "inactive" mode, at least not after 5 minutes, but after 1 ?

**Comment by RoguedBear** [comment]
I've gotten this request multiple times now (#2). I won't officially support setting a timeout of less than 5 minutes since it doesn't make sense.

however you still have a way to set it lower if you wish.

see my comment here: https://github.com/RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK/issues/2#issuecomment-1024280678
